text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
+The `url` in `resolved` is unnecessary; keeping `hash` is enough.
+
+# How We Teach This
+
+Just set the registry before `yarn install` if you do not want to use `https://registry.npmjs.org`.
+Or use `yarn install --registry=https://registry.npm.taobao.org`.
+
```

Comment:
We'll probably need to emphasize on the documentation website that `install` will use the `.yarnrc` configured registry, the command line configured registry, or will fallback to the default.